server/JsDbg.Gdb/JsDbg_test.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python
+import sys
+import unittest
+
+class GdbModule(object):
+    COMMAND_USER = 1
+
+    class Command(object):
+        def __init__(self, name, type):
+            pass
+
+sys.modules['gdb'] = GdbModule
+import JsDbg
+
+class TestJsDbg(unittest.TestCase):
```

Comment:
Would be good to add some instructions on how to run these unit tests.

Reply:
Added a comment at the top of the file. (It's just "python JsDbg_test.py")